README.md

```diff
@@ -50,7 +50,51 @@ Alternatively, to install Mapbox Navigation using [Carthage](https://github.com/
    github "mapbox/mapbox-navigation-ios" ~> 0.40
    ```
 
-1. Run `carthage update --platform iOS` to build just the iOS dependencies.
+1. Follow the rest of [Carthage’s iOS integration instructions](https://github.com/Carthage/Carthage#if-youre-building-for-ios-tvos-or-watchos). Your application target’s Embedded Frameworks should include `MapboxNavigation.framework`, `MapboxCoreNavigation.framework`, `MapboxNavigationNative.framework`, and `MapboxAccounts.framework`.
+
+## Installing the latest stable version with the Maps SDK 5.10.0-beta version
```

Comment:
There’s already a section about this below, called “Installing the latest prerelease”.

Reply:
you're right. the latest version of the branch contains it after rebasing. updated.